src/java/relex/PlainTextServer.java

```diff
@@ -122,8 +127,8 @@ else if (args[i].equals("--port") || args[i].equals("-p"))
 			}
 			else if (args[i].equals("--verbose") )
 			{
-				System.err.println("Info: Verbose server mode set.");
-				verbose = true;
+				logger.setLevel(Level.DEBUG);
+				logger.info("Info: Verbose server mode set.");
```

Comment:
I'm confused. `--verbose` is an explicit request, from the user, to print verbose output to `stdout`; dumping it to the logger makes no sense.